pkgs/desktops/cinnamon/cinnamon-desktop/default.nix

```diff
@@ -0,0 +1,67 @@
+{ fetchFromGitHub
+, gdk-pixbuf
+, gobject-introspection
+, gtk3
+, intltool
+, meson
+, ninja
+, pkgconfig
+, pulseaudio
+, python3
+, stdenv
+, xkeyboard_config
+, xorg
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cinnamon-desktop";
+  version = "4.4.0";
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = pname;
+    rev = version;
+    sha256 = "17hb8gkb9pfj56ckva5g4x83yvmdv7hvpidxjsdf79dw6pabr5rg";
+  };
+
+  buildInputs = [
+    gdk-pixbuf
+    gtk3
+    pulseaudio
+    xkeyboard_config
```

Comment:
I was wondering if there was a hardcoded path here like in https://github.com/NixOS/nixpkgs/blob/master/pkgs/desktops/pantheon/apps/switchboard-plugs/keyboard/xkb.patch but they do https://github.com/linuxmint/cinnamon-desktop/search?q=xkb_base&unscoped_q=xkb_base which is great for NixOS

Reply:
Nice